pkgs/by-name/zi/zigbee2mqtt_1/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  buildNpmPackage,
+  fetchFromGitHub,
+  systemdMinimal,
+  nixosTests,
+  nix-update-script,
+  withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemdMinimal,
+}:
+
+buildNpmPackage rec {
```

Comment:
I would rather not have the `rec` but not important xD

Reply:
The legacy package, not worth changing until it gets removed.